systemtests/python-modules/bareos_unittest/json.py

```diff
@@ -127,31 +133,37 @@ def run_restore(
             run_parameter.append("jobid={}".format(jobid))
         if fileset:
             run_parameter.append("fileset={}".format(fileset))
+            run_parameter.append("select")
```

Comment:
Why does this only need to happen when a fileset is specified ?

At least on the bconsole
```
*restore jobid=1 where=/tmp/bareos fileset=SelfTest all done yes
```
works.